### PR TITLE
Added main and options menues

### PR DESCRIPTION
**Pull request by ViktorKapra**

### CODE REVIEW

**Review by RylaD303 — CHANGES_REQUESTED** [review]
Everything seems alrighty, just do 3 small things, please:
 1. Remove the PlayerBullet changes. A bullet should not be extending a hitbox. A bullet is an entity by it's own. It should contatin a hitbox, but is not a hitbox.
 2. Please move the menu script in the script directories (options_menu.gd, Menu.gd).
 3. Preferably use pascal case for naming classes. Should be "OptionsMenu.gd". And use Snake case for scenes.

File: src/objects/mainMenu/Menu.gd [hunk 1]
```diff
@@ -0,0 +1,16 @@
+extends Control
+
+
+
```
Comment (RylaD303) [inline src/objects/mainMenu/Menu.gd:4]: Add class_name, please.

File: src/objects/mainMenu/options_menu.gd [hunk 2]
```diff
@@ -0,0 +1,6 @@
+extends Control
+
+
```
Comment (RylaD303) [inline src/objects/mainMenu/options_menu.gd:3]: Same here, add a class_name.

**Comment by ViktorKapra** [comment]
Тhe issues have been noted and fixed. Thanks for the feedback

**Review by RylaD303 — APPROVED** [review]
Well done!

**Pull request merged**